### PR TITLE
[Install] Offer an option for people that have 🐳 but not 🐍

### PR DESCRIPTION
**Pull request by gulhe**
I wanted to check and tests some things on your doc.
I have Docker, Python (pip) I don't (and not keen on installing it 😅)

So I checked, found a way, and instead of keeping it to me I wanted to share it with you.
It works like a charm on my setup, if you want it have it :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ Follow [https://www.mkdocs.org/getting-started/](https://www.mkdocs.org/getting-
...
 
 Run : 
-- `pip install mkdocs mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects`
-- `mkdocs serve` in the project root folder
-- Visit  [http://localhost:8000/](http://localhost:8000/)
+ - Via Python
+   - `pip install mkdocs mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects`
+   - `mkdocs serve` in the project root folder
+   - Visit  [http://localhost:8000/](http://localhost:8000/)
+ - Via [Docker](https://squidfunk.github.io/mkdocs-material/getting-started/ "more recent methods and docs may very well be available there") (in case you are not a big python fan)
```
Comment (KarimGl) [inline README.md:11]: May be change the docker link to https://hub.docker.com/r/squidfunk/mkdocs-material ?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ Follow [https://www.mkdocs.org/getting-started/](https://www.mkdocs.org/getting-
...
+   - build an image from de provided [Dockerfile](./Dockerfile "you'll only need to do this once") `docker build -t squidfunk/mkdocs-material .`
```
Comment (KarimGl) [inline README.md:12]: In order to not confuse with squidfunk/mkdocs-material original image, we can choose an other name for our custom image: `docker build -t chutney/doc .`

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FROM squidfunk/mkdocs-material
+RUN pip install mkdocs-material mkdocs-git-revision-date-localized-plugin mkdocs-redirects
```
Comment (KarimGl) [inline Dockerfile:2]: mkdocs-material and mkdocs-redirects are provided by squidfunk/mkdocs-material. So there is no need to install them.
`RUN pip install mkdocs-git-revision-date-localized-plugin`

**Comment by gulhe** [comment]
checking back on that soon

File: README.md (hunk 2 above)
Comment (gulhe) [reply README.md:12]: I must say on this one I fully went like what mkDocs's documentation recommends (using the same image name and so on)
Tho I get the idea of having your very own image name for clarity's sake :)

File: README.md (hunk 1 above)
Comment (gulhe) [reply README.md:11]: I'm not sure about this one, as a reader I'd much rather have a link to the mkDoc's main doc which explains all that I'm about to read along with official alternatives and a link to the dockerHub page that directly a link to the docker hub page :)

I'm not gonna change that in this PR as to leave this request how I'd like to have the doc on my consumer's perspective but you may, it's your documentation this is simply a suggestion ^^

Your kingdom your rules ;p

**Review by KarimGl — APPROVED** [review]

**Review by bessonm — APPROVED** [review]

**Comment by bessonm** [comment]
All good to me, 
  @gulhe  thanks again for your contribs :+1: 
 
On a side note, we know this project is quite confidential since we lack the time to communicate about it.
May I ask how you found about it and what are you looking for with it ?

If I may ask for a bit more of your time (to shoot a mail, zulip, call or comment),
We would be very happy to have any feedback about your opinion, usefulness and use case ! :heart:

**Pull request merged**